### ISSUE TITLE
RECAP mis-representing the cost of some documents on PACER

### ISSUE THREAD

**Issue by nk9** [freelawproject/recap#356]
RECAP describes the cost of this document at $3.00. In fact, [transcripts aren't subject to the $3/document cap](https://pacer.uscourts.gov/help/faqs/pricing). So the cost of the 179 page document in this case is actually $17.90.

<img width="946" alt="image" src="https://github.com/freelawproject/recap/assets/3646730/cf1713a4-14fc-4808-a43e-5ff4b4dd338b">

**Comment by mlissner** [comment]
Yeah, good point. I guess we could try to do a better job of figuring out what a transcript is and then showing a good price for them.
